### ISSUE TITLE
SEP 057 -- Best practices modifications on the SBOL specification

### ISSUE THREAD

**Issue by jakebeal** [SynBioDex/SEPs#125]
SEP 057 proposes a namespace for Best Practices to add terms to, so that they can use new ontology terms without having to modify the SBOL specification: https://github.com/SynBioDex/SEPs/blob/master/sep_057.md

**Comment by cjmyers** [comment]
This proposal seems reasonable to me.

**Comment by Gonza10V** [comment]
Up for vote now: https://forms.gle/YZwQtRz9W1DghXbU7

**Comment by jakebeal** [comment]
I don't think this is complete until it either rejected and withdrawn or is accepted and included in the spec.

**Comment by Gonza10V** [comment]
Ok, I'll close discussions if withdrawn or after votation. I though that now that is up for vote there is no further discussion to do.
